tests/test_associationUtils.py

```diff
@@ -0,0 +1,63 @@
+# This file is part of pipe_tasks.
+
+# Developed for the LSST Data Management System.
+# This product includes software developed by the LSST Project
+# (https://www.lsst.org).
+# See the COPYRIGHT file at the top-level directory of this distribution
+# for details of code ownership.
+
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <https://www.gnu.org/licenses/>.
+
+import healpy as hp
+import numpy as np
+import unittest
+
+from lsst.pipe.tasks.associationUtils import query_disc
+import lsst.utils.tests
+
+
+class TestAssociationUtils(lsst.utils.tests.TestCase):
+
+    def test_queryDisc(self):
+        """Test that doing a disc query of healpy using our wrapper works as
+        expected.
+
+        Most other calculations are very simple and not worth testing.
+        """
+        # Find nearest pixel to r=45, dec=45.
+        nside = 128
+        centerPixNumber = hp.ang2pix(nside, 45, 45, lonlat=True)
+        ra, dec = hp.pix2ang(nside, centerPixNumber, lonlat=True)
```

Comment:
Perhaps add a test that the healpix pixel ra and dec are less than the size of a pixel away from the initial ra and dec.

Reply:
This would be testing healpy directly vs testing the wrappers that we have written so I'm not included to add this test. If ang2pix isn't working properly there would be more fundamental problems with healpy going on.